### PR TITLE
EDGCSOFT-34 Logging improvement

### PR DESCRIPTION
**Pull request by pavankumar181**
## Purpose
EDGCSOFT-34 (https://issues.folio.org/browse/EDGCSOFT-34) 
Adding loggers as part of logging improvements

## Approach
_By following the practices mentioned in https://wiki.folio.org/display/DD/Logging,
Added loggers wherever needed._

## Evidences
image
edgcsoft-34

## Pre-Merge Checklist:
 Before merging this PR, please go through the following list and take appropriate actions.

 - Does this PR meet or exceed the expected quality standards?
   - [ ] Code coverage on new code is 80% or greater
   - [ ] Duplications on new code is 3% or less
   - [ ] There are no major code smells or security issues
 - Does this introduce breaking changes?
   - [ ] Were any API paths or methods changed, added or removed?
   - [ ] Were there any schema changes?
   - [ ] Did any of the interface versions change?
   - [ ] Were permissions changed, added, or removed?
   - [ ] Are there new interface dependencies?
   - [ ] There are no breaking changes in this PR.
   - [ ] Check logging

 If there are breaking changes, please **STOP** and consider the following:

 - What other modules will these changes impact?
 - Do JIRAs exist to update the impacted modules?
   - [ ] If not, please create them
   - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
   - [ ] Do they have all they appropriate links to blocked/related issues?
 - Are the JIRAs under active development?
   - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
 - Do PRs exist for these changes?
   - [ ] If so, have they been approved?

 Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

 While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by gurleenkaurbp — APPROVED** [review]

File: src/main/java/org/folio/ed/service/CaiaSoftIntegrationService.java [hunk 1]
```diff
@@ -57,6 +64,8 @@ public IntegrationFlowContext.IntegrationFlowRegistration registerRetrievalsPoll
   }
 
   public IntegrationFlowContext.IntegrationFlowRegistration registerRetrievalQueueRecordFlow(Configuration configuration) {
+    log.info("registerRetrievalQueueRecordFlow:: Registering retrieval queue record flow for configuration: {}", configuration.getId());
```
Comment (Vignesh-kalyanasundaram) [inline src/main/java/org/folio/ed/service/CaiaSoftIntegrationService.java:67]: Please change this to debug and keep single logger statement as method itself is having single line.

File: src/main/java/org/folio/ed/service/CaiaSoftIntegrationService.java [hunk 2]
```diff
@@ -32,18 +35,22 @@ public void updateIntegrationFlows() {
...
+    log.debug("removeExistingFlows:: Trying to remove if there are any Existing Flows");
     integrationFlowContext.getRegistry().keySet().forEach(key -> {
       integrationFlowContext.getRegistrationById(key).stop();
       integrationFlowContext.remove(key);
     });
   }
 
   public IntegrationFlowContext.IntegrationFlowRegistration registerRetrievalsPoller(Configuration configuration) {
+    log.debug("registerRetrievalsPoller:: Registering retrievals poller for configuration: {}", configuration.getId());
+    log.info("registerRetrievalsPoller:: Registered retrievals poller for configuration: {}", configuration.getId());
```
Comment (Vignesh-kalyanasundaram) [inline src/main/java/org/folio/ed/service/CaiaSoftIntegrationService.java:53]: Please remove info log and keep single logger statement as method itself is having single line.

File: src/main/java/org/folio/ed/handler/RetrievalQueueRecordHandler.java [hunk 3]
```diff
@@ -53,12 +53,16 @@ private String getCirculationUrl(Configuration configuration) {
...
+    log.debug("getCaiaSoftRequest:: Getting Caiasoft Request");
     var requestItem = CaiaSoftRequestItem.of(retrievalRecord.getHoldId(),
       retrievalRecord.getRequestType(), retrievalRecord.getItemBarcode(), retrievalRecord.getPickupLocation());
+    log.info("getCaiaSoftRequest:: Retrieved Caiasoft Request");
     return CaiaSoftRequest.of(Collections.singletonList(requestItem));
   }
 
   private ResponseEntity<String> post(String url, HttpHeaders headers, Object entity) {
+    log.debug("post:: Sending POST request to URL: {}", url);
+    log.info("post:: Sent POST request to URL: {}", url);
```
Comment (Vignesh-kalyanasundaram) [inline src/main/java/org/folio/ed/handler/RetrievalQueueRecordHandler.java:65]: Please remove info log and keep single logger statement as method itself is having single line.

**Review by Vignesh-kalyanasundaram — CHANGES_REQUESTED** [review]

File: src/main/java/org/folio/ed/service/CaiaSoftIntegrationService.java (hunk 2 above)
Comment (pavankumar181) [reply src/main/java/org/folio/ed/service/CaiaSoftIntegrationService.java]: removed info log.

File: src/main/java/org/folio/ed/service/CaiaSoftIntegrationService.java (hunk 1 above)
Comment (pavankumar181) [reply src/main/java/org/folio/ed/service/CaiaSoftIntegrationService.java]: Changed to debug.

**Review by Vignesh-kalyanasundaram — APPROVED** [review]

**Pull request merged**